src/i18n/translations/messages_en.json

```diff
@@ -47,5 +47,7 @@
   "No Internet connection": "No Internet connection",
   "Settings": "Settings",
   "Night mode": "Night mode",
-  "Language": "Language"
+  "Language": "Language",
+  "Topic": "Topic",
+  "Type a message here": "Type a message here"
```

Comment:
Good that you noticed, @saketkumar95 
I added them for him with my last commit, so now it is OK.